packages/apps/human-app/frontend/src/shared/helpers/dark-mode.ts

```diff
@@ -0,0 +1,17 @@
+const MODE_LOCAL_STORAGE_KEY = 'mode';
```

Comment:
It shouldn't be a `shared` helper since it's used only in `ColorMode` context.

It's fine to keep these functions in file with the context itself (or next to it as e.g. `color-mode-settings.ts`